include/boost/di/core/wrapper.hpp

```diff
@@ -17,7 +17,7 @@ namespace successful {
 
 template <class T, class TWrapper>
 struct wrapper {
-  inline operator T() noexcept { return __BOOST_DI_TYPE_WKND(T) wrapper_; }
+  inline operator T() noexcept { return wrapper_; }
```

Comment:
Awesome, finally :+1:

Reply:
Can we get rid of the other one in injector.create?

Reply:
Unfortunately no, I can get rid of some of them, but it still required for some `create_successful_impl`

Reply:
Oh, maybe in the future versions, though.

Reply:
Lets add each other as collaborators in our forks so that we can push commits directly in existing PRs in each other repos. Otherwise we have to create new PR each time